cfgov/unprocessed/css/organisms/featured-menu-content.less

```diff
@@ -0,0 +1,110 @@
+// Duplicated from current implementation in Capital Framework v4.6.0
+// with minor edits (to be released in CFv4.7.0)
+// This can be deleted safely when this project is ported to CFv4
+
+@fcm-bp-max: 700px;
+@fcm-bp-min: 701px;
+
+.o-featured-content-module {
+  .webfont-regular();
+
+  position: relative;
+
+  min-height: 320px;
+  margin-right: -1px;
```

Comment:
Looks like it's from the original FCM code. I think it's to fix the border+border issues 

https://github.com/cfpb/capital-framework/commit/def957a21c9f25fa745de948e673d1855ed32159#diff-5140a128023e2d3dcfd06c7e5b80e2c5L592